### ISSUE TITLE
Small typo in link

### ISSUE THREAD

**Issue by ways2read** [daisy/a11y-meta-viewer#9]
Digital is missing the 't' in "W3C Accessibility Metadata Display Guide for Digial Publications 2.0"

**Comment by mattgarrish** [comment]
Thanks, Richard, I'd stopped noticing that link!

I've got the text fixed locally, but am in the middle of adding support for the descriptive statements so it won't go live until probably later today or tomorrow.

**Comment by ways2read** [comment]
Whilst I am picking holes:

- you also need to correct the same text in the [README.md](https://github.com/daisy/a11y-meta-viewer/blob/main/README.md)
- you may want to make the references to ONIX/onix/ONIX consistent
- the link to Report Bugs could maybe point to the issue tracker

**Comment by mattgarrish** [comment]
Thanks, these have all been fixed in https://github.com/daisy/a11y-meta-viewer/commit/52645b8f38e7d1a342f4360701ba28259fe318c2
